test/e2e/alb_test.go

```diff
@@ -0,0 +1,80 @@
+// +build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/tj/assert"
+
+	"github.com/argoproj/argo-rollouts/test/fixtures"
+)
+
+type ALBSuite struct {
+	fixtures.E2ESuite
+}
+
+func TestALBSuite(t *testing.T) {
+	suite.Run(t, new(ALBSuite))
+}
+
+func (s *ALBSuite) SetupSuite() {
+	s.E2ESuite.SetupSuite()
+}
+
+const actionTemplate = `{"Type":"forward","ForwardConfig":{"TargetGroups":[{"ServiceName":"%s","ServicePort":"%d","Weight":%d},{"ServiceName":"%s","ServicePort":"%d","Weight":%d}]}}`
+
+const actionTemplateWithExperiment = `{"Type":"forward","ForwardConfig":{"TargetGroups":[{"ServiceName":"%s","ServicePort":"%d","Weight":%d},{"ServiceName":"%s","ServicePort":"%d","Weight":%d},{"ServiceName":"%s","ServicePort":"%d","Weight":%d}]}}`
+
+func (s *ALBSuite) TestALBExperimentStep() {
```

Comment:
Could you put this in `aws_test.go` instead? We don't need both aws_test.go and alb_test.go.